### ISSUE TITLE
Github-Action work flow

### ISSUE THREAD

**Issue by Elshaday-Tamire** [UBC-MDS/dsci_522_group3#21]
I created a GitHub Actions workflow file that can be tested after my current pull request is accepted. Once the current pull request is merged, I will create a new pull request for the workflow.

**Comment by meagangardner** [comment]
Thank you Elshaday! Once we have this part set up, will we be able to access our project through Docker?

**Comment by meagangardner** [comment]
It looks like the image in the docker compose file doesnt have your user name in for the image, in your docker-publish.yml workflow file, your image is set up as: 
images: elshu13/customer-term-deposits-predictor
I believe this is the way we need to link it for the image in our docker-compose.yml

**Comment by Elshaday-Tamire** [comment]
> Thank you Elshaday! Once we have this part set up, will we be able to access our project through Docker?

yes

**Comment by Elshaday-Tamire** [comment]
> It looks like the image in the docker compose file doesnt have your user name in for the image, in your docker-publish.yml workflow file, your image is set up as: images: elshu13/customer-term-deposits-predictor I believe this is the way we need to link it for the image in our docker-compose.yml

The line images: elshu13/customer-term-deposits-predictor is part of the workflow file( which is my docker hub username), not the Docker Compose file. Our image name is simply customer-term-deposits-predictor.

**Comment by Elshaday-Tamire** [comment]
I have created a new pull request for the GitHub Actions workflow.

**Comment by meagangardner** [comment]
It looks like the line images: elshu13/customer-term-deposits-predictor should also be part of the docker-compose.yml file as well though, as seen in tiffany's example: 
image:  ttimbers/breast-cancer-predictor:e8eb36e
This also contains a tag at the end, which is found on dockerhub

**Comment by Elshaday-Tamire** [comment]
Oh, I see what you're saying. We're planning to do that after we have the image on Docker Hub. Once the GitHub Actions workflow is merged into the main branch and our Docker image is published on Docker Hub, we can update the Docker Compose file with the image name and username from Docker Hub.

What I’ve done in the Dockerfile is for you to test by creating the Docker image on your machine. You can use the docker-compose up command to run it and test the analysis file the same way I did on my side.

**Comment by Elshaday-Tamire** [comment]
Hi team. Workflow merged to main and docker-compose file is ready to use.
Thank you

**Comment by meagangardner** [comment]
I was able to run docker-compose up on my computer and run the file - thank you!
